docs/sidebars.js

```diff
@@ -366,20 +355,34 @@ module.exports = {
         },
         {
           type: "doc",
-          id: "troubleshooting/troubleshooting-re",
+          id: "troubleshooting/troubleshootingre",
```

Comment:
why this change?

Reply:
The ID "troubleshooting" already exists for Troubleshooting page of Local Engine. hence, we've created a new ID "troubleshootingre" for the Troubleshooting page of Replicated Engine.